docs/source/release_notes.rst

```diff
@@ -99,8 +99,8 @@ Changes
 -------
 
 - Passing :term:`python:coroutine`\ s to :func:`~cocotb.triggers.with_timeout` is now supported. (:pr:`2494`)
-- Renamed ``RunningTask`` to :class:`~cocotb.decorators.Task`. (:pr:`2876`)
-- Made :class:`~cocotb.decorators.Task` interface more like :py:class:`asyncio.Task`'s. (:pr:`2876`)
+- Renamed ``RunningTask`` to :class:`~cocotb.task.Task`. (:pr:`2876`)
+- Made :class:`~cocotb.task.Task` interface more like :py:class:`asyncio.Task`'s. (:pr:`2876`)
```

Comment:
```suggestion
- Made :class:`~cocotb.task.Task` interface more like :python:class:`asyncio.Task`'s. (:pr:`2876`)
```
This is what our intersphinx mapping uses, https://github.com/cocotb/cocotb/blob/32b8581c8434fbadf2ea6ce35cd97e44b9fc9a45/docs/source/conf.py#L52C6-L52C12, but I think strictly speaking, you would not need it at all.